src/capi.ts

```diff
@@ -13,7 +13,7 @@ import { fromString as dateFromString } from 'date';
 import type { MainMedia } from 'headerMedia';
 import { MainMediaKind } from 'headerMedia';
 import { parseImage } from 'image';
-import { isAdvertisementFeature } from 'item';
+import { isSpecialLabs } from 'item';
```

Comment:
We should make sure everything is referring to `Labs` rather than `SpecialLabs` throughout.

ie this should be `isLabs`